### DISCUSSION TITLE
Open3d 0.15.2 vs 0.15.1 - windows vs linux

### DISCUSSION THREAD

**Discussion by Koshmaar** [isl-org/Open3D#5189] [General]
Hey, using pip I installed open3d on my linux (ubuntu 20) and windows 11 machines. Results:
* linux got 0.15.2
* windows got 0.15.1

For some reason windows doesn't find 0.15.2

I can see tag 0.15.1 in this github repo (from 19 Feb), but I can't see tag 0.15.2. However, there's pip release 0.15.2 on pypi (https://pypi.org/project/open3d/0.15.2/#history) from 28 Feb. There's distinction between results I'm getting on those two machines, and I suspect something could change in implementation of some algorithm. 

I'd like to confirm that, so I'm looking for changelog between those two versions, and can't find anything. All I could find is the big one https://github.com/isl-org/Open3D/releases/tag/v0.15.1 While there's none for the minor versions. 

My two questions:
1. Is pypi release 0.15.2 coming from master on github?
2. Is there changelog with details about changes between minor versions? 

Thanks

**Comment by yuecideng** [comment]
This [link](https://github.com/isl-org/Open3D/pull/4817) might be helpful.

**Reply by Koshmaar** [reply]
Ok, thanks. So indeed 15.2 is with fix only for linux. However, this version could be also uploaded with no changes for windows, so that there is uniformity between platforms. Having it like it is now, is confusing for developers and makes it harder to built reliable CI scripts.
